screens/home/index.html

```diff
@@ -1,14 +1,41 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="pt-BR">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Home</title>
+    <link rel="icon" href="../../assets/images/navigation.svg">
+    <title>Navegação</title>
+    <!-- Fontes (Google Fonts) -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&display=swap" rel="stylesheet">
+    <!-- Bootstrap -->
+    <link href="../../assets/css/bootstrap.css" rel="stylesheet">
+    <script src="../../js/bootstrap.bundle.js" defer></script>
+    <!-- Vanilla CSS -->
+    <link href="../../assets/css/navigation.css" rel="stylesheet">
 </head>
 <body>
-    
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <div class="container-fluid min-vh-100 d-flex flex-column">
+        <div class="row flex-grow-1">
+            <div class="col-2 d-block sidebar">
+                <input class="img-button" type="image" src="../../assets/images/back.svg" />
+                <input class="img-button" type="image" src="../../assets/images/agenda.svg" />
+                <input class="img-button" type="image" src="../../assets/images/journey.svg" />
```

Comment:
Isso aqui deve ser uma tag <a> que leva pra tela que eu estou construindo.

Para que você já possa criar o link, o meu arquivo está em screens/list-of-rotines/index.html . É só você jogar o href da tag anchor pra lá